ssl/t1_enc.c

```diff
@@ -563,7 +563,7 @@ int tls1_setup_key_block(SSL *s)
     s->s3.tmp.key_block = p;
 
     OSSL_TRACE_BEGIN(TLS) {
-        BIO_printf(trc_out, "key block length: %ld\n", num);
+        BIO_printf(trc_out, "key block length: %zd\n", num);
```

Comment:
I can see quite a few other places using %zu in the code..

This appears to be ANSI C99 though.

Reply:
It's out function, `BIO_printf` so the C version isn't important.
I've switched it to be unsigned.